src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String email;
+    private String password;
+    private byte[] salt;
+
+    public User() {
+    }
```

Comment:
you shouldn't create this constructor explicitly in case if you don't create new ones, so remove it. I agree it confuses and we'll fix this very soon